runtime/doc/eval.txt

```diff
@@ -1641,6 +1641,7 @@ v:event		Dictionary of event data for the current |autocommand|.  Valid
 					|v:false| if not.
 			changed_window 	Is |v:true| if the the event fired
 					while changing window (or tab) on |DirChanged|.
+			status		Exit code of the job in a terminal on |TermClose|.
```

Comment:
Could be used for other contexts, not only terminal. Let's leave the door open for that.
```suggestion
			status		Job status or exit code. |TermClose|
```